kernelci/cli/base.py

```diff
@@ -178,6 +178,11 @@ class Args:  # pylint: disable=too-few-public-methods
         'help': "Path to the dtbs.json file",
     }
 
+    group_id = {
+        'name': '--group-id',
```

Comment:
How about just using it as a positional argument `group_id` like for `kci node get`?

Reply:
Yes, updated it.